sdk/spring/azure-spring-boot/HEALTH_INDICATOR_README.md

```diff
@@ -0,0 +1,110 @@
+# Azure Spring Boot - Actuator
```

Comment:
How about adding this doc as a section in this [README.md](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/README.md).

Reply:
And I don't prefer the name of Azure Spring Boot Actuator, for we don't have a separate actuator module. And the Spring Boot actuator module contains not only health information.